integration-cli/docker_cli_swarm_test.go

```diff
@@ -1186,7 +1186,7 @@ func (s *DockerSwarmSuite) TestSwarmLockUnlockCluster(c *check.C) {
 	c.Assert(getNodeStatus(c, d4), checker.Equals, swarm.LocalNodeStateActive)
 }
 
-func (s *DockerSwarmSuite) TestSwarmJoinPromoteLocked(c *check.C) {
+func (s *DockerSwarmSuite) TestSwarmJoinPromoteLocked_FLAKY(c *check.C) {
```

Comment:
Added a temporary commit to trigger our new "flaky check" 🎉 @olljanat 🤞

Reply:
@thaJeztah integration-cli/ is deprecated so flaky check is only looking for tests under integration/ folder...

How ever you can easily test that old test on locally by running commands:
```
export TEST_REPEAT="5"
export TIMEOUT="30m"
export TESTFLAGS="-test.count 5 -check.f TestSwarmJoinPromoteLocked"
make test-integration
```

Reply:
LOL, yes, just thought of that hahaha


```
19:15:21 The following new tests were added to integration-cli:

19:15:21 +func (s *DockerSwarmSuite) TestUnlockEngineAndUnlockedSwarm_FLAKY(c *check.C) {
19:15:21 +func (s *DockerSwarmSuite) TestSwarmJoinPromoteLocked_FLAKY(c *check.C) {
19:15:21 
19:15:21 integration-cli is deprecated. Please add an API integration test to
19:15:21 ./integration/COMPONENT/. See ./TESTING.md for more details.
```

Reply:
But you can of course also set those to https://github.com/moby/moby/blob/37498f009d8bf25fbb6199e8ccd34bed84f2874b/hack/make/.integration-test-helpers#L14-L15 if you want stress test old test with CI.